robot/Cumulus/resources/locators_51.py

```diff
@@ -325,6 +325,10 @@
         'remove_lookup':'//div[@class="modal-body scrollable slds-modal__content slds-p-around--medium"]//*[contains(@data-qa-locator,"button Remove selected option Opportunity: Record Type")]',
         'click-dropkown-value':"//div[@class='modal-body scrollable slds-modal__content slds-p-around--medium']//*[contains(@class,'slds-combobox')]//span[text()='{}'']",
         'modal_lookup_button':'//div[contains(@class,"modal-body")]//*[contains(@data-qa-locator,"{}")]//child::button',
+        'temp_builder_labels':'//lightning-input[@class="slds-truncate slds-form-element"]//label',
+        'ge_form_labels':'//label[@class="slds-form-element__label" or @class="slds-form-element__label slds-no-flex"]',
+        'temp_builder_sections':'//h2[@class="slds-card__header-title"]',
+        'ge_form_sections':'//span[contains(@title, "FormSection")]'
```

Comment:
Instead of hardcoding the title here you can leave it as a placeholder //span[contains(@title, "{}")] and send it from the code.

Reply:
Fixed!